INSTALL.rst.txt

```diff
@@ -81,12 +78,8 @@ recent gfortran from an OS X package manager (like Homebrew).
 Please do NOT use gfortran from `hpc.sourceforge.net <http://hpc.sourceforge.net>`_,
 it is known to generate buggy scipy binaries.
 
-
-Blas/Lapack
-~~~~~~~~~~~
-
-Mac OS X includes the Accelerate framework: it should be detected without any
-intervention when building SciPy.
+You should also use a BLAS/LAPACK library from an OS X package manager.
+ATLAS, OpenBLAS, and MKL all work.
```

Comment:
It would be good to have something more explicit here on the lines of "As of Scipy version X, we do not support compiling against the system Accelerate library for BLAS / LAPACK, because it does not support the full range of LAPACK routines."